tests/upgrades/test_satellitesync.py

```diff
@@ -0,0 +1,138 @@
+"""Test for Inter Satellite Sync related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: InterSatelliteSync
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from fauxfactory import gen_string
+from nailgun import entities
+from robottelo import ssh
+from robottelo.cli.contentview import ContentView
+from robottelo.cli.package import Package
+from upgrade_tests import pre_upgrade, post_upgrade
+from robottelo.test import CLITestCase
+from upgrade_tests.helpers.scenarios import create_dict, get_entity_data
+
+
+class scenario_preversion_cv_exports_imports(CLITestCase):
+    """Test Content-view created before upgrade can be exported and imported after upgrade
+
+        :id: f19e4928-94db-4df6-8ce8-b5e4afe34258
```

Comment:
As in upgrade we not using `ID` as class level . I am not sure how I missed this :) 
Could you please update the doc string like https://github.com/SatelliteQE/robottelo/blob/master/tests/upgrades/test_client.py#L238-L248

Reply:
Ha bhai ! kar diya :)

By the way, ```:steps:``` is correct as per testimony, but this change is required in all upgrade scenarios. Will raise a separate PR.